csrc/batch_prefill_customize_config.jinja

```diff
@@ -108,6 +112,10 @@ struct PagedParams {
   uint32_t* total_num_rows;
   uint32_t padded_batch_size;
   bool partition_kv;
+  uint32_t* prefix_len_ptr;
```

Comment:
Can you move them to additional params? I tend to managing all of the options parameters as additional, instead of default ones, which is easier to manager.

Examples include: https://github.com/flashinfer-ai/flashinfer/blob/9220fb3443b5a5d274f00ca5552f798e225239b7/flashinfer/jit/attention/pytorch.py#L456 (we use prefix `maybe_` to indicate these components are optional and have type std::optional<...>).

Reply:
@yzh119 as suggested, moved multi-item scoring parameters as addtional.